test/acceptance/features/postgresqlOperatorExamples.feature

```diff
@@ -0,0 +1,331 @@
+                    version: v1alpha1
+                    kind: Database
+                    name: db-demo-1
+            """
+        When Service Binding is applied
+            """
+            apiVersion: binding.operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-2
+            spec:
+                application:
+                    name: nodejs-app
+                    group: apps
+                    version: v1
+                    resource: deployments
+                services:
+                -   group: postgresql.baiju.dev
+                    version: v1alpha1
+                    kind: Database
+                    name: db-demo-2
+            """
+        Then Service Binding "binding-request-1" is ready
+        And Service Binding "binding-request-2" is ready
+        And "nodejs-app" deployment must contain SBR name "binding-request-1"
+        And "nodejs-app" deployment must contain SBR name "binding-request-2"
+
+    # https://github.com/redhat-developer/service-binding-operator/tree/master/examples/pod_spec_path
+    Scenario: Specify container's path in Service Binding
+        Given DB "db-demo-csp" is running
+        * The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: AppConfig
+            metadata:
+                name: demo-appconfig-csp
+            spec:
+                uri: "some uri"
+                Command: "some command"
+                image: my-image
+                spec:
+                    containers:
+                    - name: hello-world
+                      # Image from dockerhub, This is the import path for the Go binary to build and run.
+                      image: yusufkaratoprak/kubernetes-gosample:latest
+                      ports:
+                      - containerPort: 8090
+            """
+        When Service Binding is applied
+            """
+            apiVersion: binding.operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-csp
+            spec:
+                application:
+                    name: demo-appconfig-csp
+                    group: stable.example.com
+                    version: v1
+                    resource: appconfigs
+                    bindingPath:
+                        containersPath: spec.spec.containers
+                services:
+                  - group: postgresql.baiju.dev
+                    version: v1alpha1
+                    kind: Database
+                    name: db-demo-csp
+                    id: zzz
+            """
+        Then Service Binding "binding-request-csp" is ready
+        And Secret "binding-request-csp" has been injected in to CR "demo-appconfig-csp" of kind "AppConfig" at path "{.spec.spec.containers[0].envFrom[0].secretRef.name}"
+
+    Scenario: Specify secret's path in Service Binding
```

Comment:
same as above.

Reply:
Yes we've at https://github.com/redhat-developer/service-binding-operator/pull/880/files#diff-ca848b998fbd78a0989748822c09065858004a5e79b457ae9b57ad5ed9dcd25cR117